server/src/authority/authority.rs

```diff
@@ -520,7 +520,7 @@ impl Authority {
                                                               true);
 
                     signer
-                        .verify_message(update_message, sig.sig())
+                        .verify_message(update_message, sig.sig(), sig)
```

Comment:
This looks like a pattern. I wonder if `verify_message` should be changed to only take the `SIG` parameter, and extract the signature inside the function:
```
    pub fn verify_message(&self,
                          message: &Message,
                          sig0: &SIG)
                          -> DnsSecResult<()> {
        let signature = sig0.sig();
        self.hash_message(message, sig0)
            .and_then(|hash| self.verify(&hash, signature))
    }

```